### DISCUSSION TITLE
Hamburger menu not working

### DISCUSSION THREAD

**Discussion by danieltorridon** [twbs/bootstrap#40777] [Q&A] [answered]
Why does my hamburger menu not work when I include the following tags...

In the HEAD...

`<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet">`

Just before the BODY...

`<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>`

`<script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js" integrity="sha384-IQsoLXl5PILFhosVNubq5LC7Qb9DXgDA9i+tQ8Zj3iwWAwPtgFTxbJ8NT4GN1R8p" crossorigin="anonymous"></script>`

`<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.min.js" integrity="sha384-cVKIPhGWiC2Al4u+LWgxfKTRIcfu0JTxR+EQDz/bgldoEyl4H0zUF0QKbrJ0EcQF" crossorigin="anonymous"></script>`

However, when I swap the last tag out for this, it works fine...

`<script src="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>`

The version 5 documentation says to include the 5.0.2 distribution in the <head> even though the current version is 5.3.0. I've tried that but it makes no odds. I've also tried different version numbers in the </body> but only version 4.0.0 works. 

Also, why am I including the bootstrap.min.js twice? Why not just in the HEAD?

And what's with the integrity property?

Some of the documentation tells me to use this in the <head>...

`<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet">`

Other places tell me to use this...

`<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">`

If I change the version of the one with the integrity property it breaks, so I'm assuming it's there to verify something? However, the documentation for the 5.3.0 version doesn't have the integrity property.

In summary, if I want the latest version of Bootstrap, and I want the hamburger menu to work, what tags should I be using in the HEAD and BODY please?

Thanks in advance!

**Comment by Alex-Toucan** [comment, marked as answer]
Hello, I recommend upgrading to v5.3.0 by using https://getbootstrap.com/docs/5.3/getting-started/introduction/.

If that's not possible, you could use the bundled JavaScript file with both Bootstrap and Popper included: `<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>`.

Also, do you have a code example that you could give us?

**Reply by danieltorridon** [reply]
Hey Alex, thank you... I found I was trying to using the 5.3 distro code with the 4.0 nav code. Didn't realise the nav component had changed in 5.3. The specific documentation for 5.3 was exactly what I needed. Many thanks :)

**Reply by Alex-Toucan** [reply]
You're welcome @danieltorridon.
